contents/blog/posthog-vs-launchdarkly.md

```diff
@@ -0,0 +1,566 @@
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        </tr>
+        <tr>
+            <td>Free collaboration</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+PostHog’s feature flag pricing is pay-per-request (and A/B tests use feature flags). There is a generous free tier of 1M requests per month with all features, add-ons, and integrations available.
+
+![PostHog pricing](../images/blog/posthog-vs-launchdarkly/posthog-price.png)
+
+LaunchDarkly’s pricing is more complicated. They charge by seat starting at $10/seat/month. On top of this, they charge based on "monthly context instances" (MCIs) also known as monthly active users. The pro plan only gets 1,000 frontend MCIs per month.
+
+![LaunchDarkly pricing](../images/blog/posthog-vs-launchdarkly/launchdarkly-price.png)
+
+As mentioned earlier, many of the security, approval, and workflow features that differentiate them are only available at the opaque "Enterprise" level. 
+
+### Reporting
+
+LaunchDarkly features basic reporting but requires other platforms for more advanced analysis. PostHog builds in advanced analysis like SQL querying, conversion funnels, and dashboards and combines flag data with other product data.
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td>PostHog</td>
+            <td>LaunchDarkly</td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Analytics</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Capture usage and calls related to flags</td>
+        </tr>
+        <tr>
+            <td>Basic visualizations</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Trends, conversion.</td>
+        </tr>
+        <tr>
+            <td>Advanced visualizations</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Funnels, retention.</td>
+        </tr>
+        <tr>
+            <td>Dashboard</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Analytics for a flag</td>
+        </tr>
+        <tr>
+            <td>Session replays</td>
```

Comment:
Two things: 

1. You repeat dashboards as an entry twice (line 367 and 349). 
2. I think session replays and SQL are worth separating out. 

Replays doesn't really fall into the reporting bucket and could be worth including a broad section about the wider toolset. The way I think I'd approach this is with a section at the start about the platforms as a whole, which visually clarifies the major features -- FFs, replays, analytics, surveys, etc. Really ram home at the start that LD if a FF tool, PostHog is more than that. 

Since you draw a line between simple/advanced visualizations (and I draw a line between reporting/analytics) I think that's worth drawing out into it's own section too -- maybe called `reporting vs analytics`. You could spell out that LD and PH have some basic insights for tracking FF performance, but go on to clarify that PH goes much farther with SQL, formulas, and additional insights.

Reply:
Updated to add more depth on the differences and features between the two